src/serac/physics/materials/tests/thermomechanical_material.cpp

```diff
@@ -0,0 +1,303 @@
+#include <iostream>
+
+#include <gtest/gtest.h>
+
+#include "serac/numerics/functional/tensor.hpp"
+#include "serac/numerics/functional/tuple.hpp"
+#include "serac/numerics/functional/tuple_arithmetic.hpp"
+
+
+namespace serac {
+
+static constexpr auto I = Identity<3>();
+
+// An arbitrary rotation tensor for frame indifference tests
+static const tensor<double, 3, 3> Q{{{ 0.852932108456964,  0.405577416697288, -0.328654495524269 },
+                                     {-0.51876824611259 ,  0.728725283562492, -0.44703351990878  },
+                                     { 0.058192140263313,  0.551784758906877,  0.831953877717782 }}};
+
+/**
+ * @brief Compute Green's strain from the displacement gradient
+ */
+template <typename T>
+auto green_strain(const tensor<T, 3, 3>& grad_u)
```

Comment:
We try to use `camelCase` for function names. See [here](https://serac.readthedocs.io/en/latest/sphinx/dev_guide/style_guide.html).

Reply:
Fixing that. Does this apply to the methods in the material model struct as well?

Reply:
Yes - classes are Pascal and methods are camel.